File: template-application/src/main/java/com/cynricshu/Application.java
package com.cynricshu;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration;

/**
 * ClusterConsole
 *
 * @author Xiong Chao(xiongchao@baidu.com)
 */
@SpringBootApplication(scanBasePackageClasses = Application.class, exclude = ErrorMvcAutoConfiguration.class)
public class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: template-application/src/main/kotlin/com/cynricshu/Application.kt
// Copyright (C) 2018 Baidu Inc. All rights reserved.

package com.cynricshu

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/**
 * Application
 *
 * @author Shu Lingjie(shulingjie@baidu.com)
 */
@SpringBootApplication(scanBasePackageClasses = [Application::class])
open class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}